network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -309,6 +309,25 @@ def __str__(self):
         return self.page.title + '->' + self.highlight.title
 
 
+class HomepageNewsYouCanUse(WagtailOrderable, models.Model):
```

Comment:
It feels like this is generic enough for the homepage (which has several sections that all tap into the blog posts) that it might be worth creating an abstract superclass here, and have all the "it's another {`page`, `blog`, `panels`}" models inherit from that?

Reply:
I would vote for having a superclass since the two classes `HomepageNewsYouCanUse` and `HomepageFeaturedBlogs` seem pretty much identical. Wanna file a follow-up?

Reply:
https://github.com/mozilla/foundation.mozilla.org/issues/4963

Reply:
```suggestion
class HomepageNewsYouCanUse(WagtailOrderable):
```